src/ch02/ch02-01_Emacs.md

```diff
@@ -0,0 +1,35 @@
+## Emacs
+
+[SLIME](https://github.com/slime/slime/) é o Superior Lisp
+Interaction Mode for Emacs. Ele possui suporte para interação com um processo
+de Common Lisp rodando, para compilação, debugging, documentação, etc.
+Funciona com diferentes implementações.
+
+[Portacle](https://shinmera.github.io/portacle/) é um ambiente Common Lisp
+multiplatforma e portável (não é preciso instalar binários na máquina).
+Vem com Emacs25, SBCL, Quicklisp, Slime e Git.
+
+<img src="../../assets/portacle.png"
+     style="width: 800px"/>
+
+### Instalando Slime
+
+Slime está no repositório oficial para pacotes Emacs Lisp, GNU ELPA (do Emacs24 em diante).
+Instale-o com:
+
+	  M-x package-install RET slime RET
+
+Agora você pode rodar Slime com `M-x slime`.
+
+Veja também:
+
+* http://wikemacs.org/wiki/SLIME - exemplos de configuração e extensões (em inglês).
+
+### Usando Emacs como uma IDE.
+
+TODO: WTF? SEPARARAM PQ?
```

Comment:
Imagino que seja interessante retirar este TO-DO aqui. Não sei se dá pra deixar comentário em Markdown mas, se tiver como, poderia ser deixado como comentário. Se for um caso futuro de a gente reorganizar o cookbook em geral sem seguir o original à risca, talvez a parte do Emacs entre aqui.

Ah, imagino que tenha sido separado porque configurar o resto do Emacs poderia ser um trabalho grande demais para uma página só.

Reply:
Esses todo eram como eu me organizava. Pra deixar pra depois ou perguntar no CLBR.
Não era pra ter mais nenhum a esse ponto